config/var/dev.cson

```diff
@@ -1,6 +1,6 @@
 {
 	mongo:
-		host: "mongodb://mongo"
```

Comment:
Just FYI: originally it was configured to work in a Docker-based dev setup
However "containerless" development seems more convenient, so yeah, let's change it, I guess.